src/viewer.js

```diff
@@ -1680,6 +1680,17 @@ class VolumeImageViewer {
       }
 
       clickEvent = 'click'
+      const features = this[_map].getFeaturesAtPixel(event.pixel)
+
+      if (features.length === 0) {
+        console.info('No feature clicked.')
```

Comment:
ROI selected should only be published when selected, if we merge this we break this event expectations for other users. It's better to cleanup state before every click than using a no click event to cleanup.

Reply:
I still need to receive an event from dicom-microscopy-viewer when the map is clicked and no ROI is selected. Since using the current event isn't ideal, I created a separate one specifically for this case.

Reply:
Instead of no roi selected, expose a viewport click event which does not use roi handlers. Just listen to map event and expose it through a viewport event. You need to get the rois and provide through the event details. if rois.length === 0 *check done in the client side then you have no rois clicked.

Reply:
Please check if what I did now is what you meant.